packages/components/form/src/types.ts

```diff
@@ -22,6 +22,7 @@ export interface FormItemRule extends RuleItem {
 }
 
 type Primitive = null | undefined | string | number | boolean | symbol | bigint
+type BrowserNativeObject = Date | FileList | File | Blob
```

Comment:
Maybe it should also include `RegExp`.

Reply:
I've considered adding `RegExp` before, but I don't think anyone would submit a `RegExp`... Should I add?

Reply:
Okay.